src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -7007,6 +7007,63 @@ def test_node_public_ip_tags(self, resource_group, resource_group_location):
             self.is_empty(),
         ])
 
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='eastus')
+    def test_vms_agentpool_type(self, resource_group, resource_group_location):
```

Comment:
Queued [live test pipeline](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=77537491&view=results) to validate the change.